test/parallel/test-crypto-classes.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+if (!common.hasCrypto) {
```

Comment:
Fips fail: `# Error: crypto.createCipher() is not supported in FIPS mode.`

You probably want to add a check against `common.hasFipsCrypto`